### PR TITLE
[MIG] website_sale_product_attribute_filter_visibility: Migration to 14.0

### PR DESCRIPTION
**Pull request by ioans73**
Standard migration to 14.0

### CODE REVIEW

**Review by ofonseca-pyming — APPROVED** [review]
Reviewed and tested

File: website_sale_product_attribute_filter_visibility/README.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    :target: https://odoo-community.org/page/development-status
+    :alt: Production/Stable
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fe--commerce-lightgray.png?logo=github
+    :target: https://github.com/OCA/e-commerce/tree/13.0/website_sale_product_attribute_filter_visibility
+    :alt: OCA/e-commerce
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-13-0/e-commerce-13-0-website_sale_product_attribute_filter_visibility
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_visibility/README.rst]: Please regenerate the readme as well

Comment (ioans73) [reply website_sale_product_attribute_filter_visibility/README.rst]: Done

**Review by Abranes — APPROVED** [review]

**Pull request merged**